jsapp/js/account/usage/usageProjectBreakdown.tsx

```diff
@@ -1,11 +1,198 @@
+
+const ProjectBreakdown = () => {
+  const [isActiveBack, setIsActiveBack] = useState(false);
+  const [isActiveForward, setIsActiveForward] = useState(false);
+  const [isComponentMounted, setIsComponentMounted] = useState(true);
+
+  const handleClick = (
+    event: React.MouseEvent<HTMLButtonElement>,
+    buttonType: ButtonType
+  ): void => {
+    event.preventDefault();
+    setIsActiveBack(buttonType === 'back');
+    setIsActiveForward(buttonType === 'forward');
+  };
+
+  const [projectData, setProjectData] = useState<Project>({
+    count: '0',
+    next: null,
+    previous: null,
+    results: [],
+  });
+
+  const onFetchAssetsDone = (data: Project) => {
+    if (isComponentMounted) {
+      const updatedResults = data.results.map((projectResult) => {
+        const assetParts = projectResult.asset.split('/');
+        const uid = assetParts[assetParts.length - 2];
+        return {
+          ...projectResult,
+          uid: uid,
+        };
+      });
+
+      setProjectData((prevData) => {return {...prevData, results: updatedResults};});
+    }
+  };
+
+ const onFetchAssetsDataDone = (assetsData: any, updatedResults: ProjectResult[]) => {
+  if (isComponentMounted) {
+    const updatedData = updatedResults
+      .filter((projectResult) => {
+        const matchingAsset = assetsData.results.find(
+          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
+        );
+
+        // Include the asset in the updatedData only if it is a survery
+        return matchingAsset?.asset_type === 'survey';
+      })
+      .map((projectResult) => {
+        const matchingAsset = assetsData.results.find(
+          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
+        );
+
+        if (matchingAsset) {
+          return {
+            ...projectResult,
+            assets: [matchingAsset],
+            deployment_status: matchingAsset.deployment_status,
+          };
+        }
+        return projectResult;
+      });
+
+    setProjectData((prevData: Project) => {
+      return {
+        ...prevData,
+        results: updatedData as ProjectResult[],
+      };
+    });
+  }
+};
+
+  // This code will be updated to only have one api call when the asset_usage api is updated
+  // Right now, you may need to refresh or re-navigate to the page multiple times as the Loading Spinner can be shown indefinitely
+  useEffect(() => {
+    const usageUrl = `${ROOT_URL}/api/v2/asset_usage`;
+
+    const fetchData = $.ajax({
+      dataType: 'json',
+      method: 'GET',
+      url: usageUrl,
+    })
+      .done(onFetchAssetsDone)
+      .then(() => {
+        const assetsUrl = `${ROOT_URL}/api/v2/assets`;
+
+        return $.ajax({
+          dataType: 'json',
+          method: 'GET',
+          url: assetsUrl,
+        })
+          .done((assetsData) =>
+            onFetchAssetsDataDone(assetsData, projectData.results)
+          )
+          .fail(onAnyFail);
+      })
+      .fail(onAnyFail);
```

Comment:
Just noting that we're trying to avoid using jQuery in new code unless it's necessary. It's fine right here - I understand that this is temp code that's being copied from older stuff 😄 This is just a gentle reminder that when you do update it later, it should use `fetchGet()` and live in `usage.api.ts`. 👍